DasharoPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c

```diff
@@ -1686,10 +1686,17 @@ PlatformBootManagerAfterConsole (
   //
   if (FUMEnabled) {
     DEBUG((DEBUG_INFO, "Registering iPXE boot option for FUM\n"));
-    SyncFvBootOption (PcdGetPtr (PcdiPXEFile),
-                      (CHAR16 *) PcdGetPtr(PcdiPXEOptionName),
-                      TRUE,
-                      FALSE);
+    if (PcdGetBool(PcdDoNotBootDtsInFUM)) {
+      SyncFvBootOption (PcdGetPtr (PcdiPXEFile),
+                        (CHAR16 *) PcdGetPtr(PcdiPXEOptionName),
+                        FALSE,
+                        FALSE);
+    } else {
+      SyncFvBootOption (PcdGetPtr (PcdiPXEFile),
+                        (CHAR16 *) PcdGetPtr(PcdiPXEOptionName),
+                        TRUE,
+                        FALSE);
+    }

```

Comment:
This is simpler version:
```suggestion
    SyncFvBootOption (PcdGetPtr (PcdiPXEFile),
                      (CHAR16 *) PcdGetPtr(PcdiPXEOptionName),
                      !PcdGetBool(PcdDoNotBootDtsInFUM),
                      FALSE);
```